src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -163,6 +171,17 @@ public static FlowExecutionList get() {
         return l;
     }
 
+    /**
+     * Returns true if all executions that were present in this {@link FlowExecutionList} have been loaded and resumed.
+     *
+     * This takes place slightly after {@link InitMilestone#COMPLETED} is reached during Jenkins startup.
```

Comment:
See https://github.com/jenkinsci/jenkins/blob/960925fc5055208d2b64a0388161ddcbcd5916aa/core/src/main/java/jenkins/model/Jenkins.java#L987-L1033. Perhaps we could instead add an API to core to indicate that `Jenkins.<init>` is truly complete, although that would not help us right at this moment, or maybe we could check for `STARTUP_MARKER_FILE` instead or something.

Reply:
Another idea from @jglick is to refactor `FlowExecutionList$ItemListenerImpl.onLoaded` to instead be an `@Initializer` method so that we can then just use `InitMilestone.COMPLETED`.

Reply:
See https://github.com/jenkinsci/workflow-api-plugin/pull/188#issuecomment-996028895 and later comments. For now we plan to leave this method in place but mark it as `@Restricted(Beta.class)`.

Reply:
I like the idea of using STARTUP_MARKER_FILE since it's already there, but this way seems more explicit without having to dig into core